NorthstarDLL/logging/logging.cpp

```diff
@@ -220,3 +222,94 @@ void NS::log::FlushLoggers()
 
 	spdlog::default_logger()->flush();
 }
+
+// Wine specific functions
+typedef const char *(CDECL *wine_get_host_version_type)(const char**, const char**);
+wine_get_host_version_type wine_get_host_version;
+
+typedef const char *(CDECL *wine_get_build_id_type)(void);
+wine_get_build_id_type wine_get_build_id;
+
+typedef char *(CDECL *wine_get_unix_file_name_type)(LPCWSTR);
+wine_get_unix_file_name_type wine_get_unix_file_name;
+
+typedef WCHAR *(CDECL *wine_get_dos_file_name_type)(LPCSTR);
+wine_get_dos_file_name_type wine_get_dos_file_name;
+
+void StartupLog()
+{
+	spdlog::info("NorthstarLauncher version: {}", version);
+	spdlog::info("Command line: {}", GetCommandLineA());
+	spdlog::info("Using profile: {}", GetNorthstarPrefix());
+
+	HMODULE ntdll = GetModuleHandleA("ntdll.dll");
+	if(!ntdll)
+	{
+		// How did we get here
+		spdlog::info("Operating System: Unknown");
+		return;
+	}
+
+	wine_get_host_version = (wine_get_host_version_type)GetProcAddress(ntdll, "wine_get_host_version");
+	if (wine_get_host_version)
+	{
+		// Load the rest of the functions we need
+		wine_get_build_id = (wine_get_build_id_type)GetProcAddress(ntdll, "wine_get_build_id");
+    	wine_get_unix_file_name = (wine_get_unix_file_name_type)GetProcAddress(GetModuleHandleA("KERNEL32"), "wine_get_unix_file_name");
+    	wine_get_dos_file_name = (wine_get_dos_file_name_type)GetProcAddress(GetModuleHandleA("KERNEL32"), "wine_get_dos_file_name");
+
+		const char* sysname;
+		wine_get_host_version(&sysname, NULL);
+
+		spdlog::info("Operating System: {} (Wine)", sysname);
+		spdlog::info("Wine build: {}", wine_get_build_id());
+
+		char* compatToolPtr = std::getenv("STEAM_COMPAT_TOOL_PATHS");
```

Comment:
`GetEnvironmentVariable` would be more winapi oriented, but getenv doesn't require us to manually deal with stuff, plus its standard.